### PR TITLE
Fixed sorting, now case-insensitive

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**

### CODE REVIEW

**Comment by ge0ffrey** [comment]
jenkins retest this

**Comment by ge0ffrey** [comment]
06:36:49 Caused by: org.codehaus.cargo.container.ContainerException: Deployable [http://localhost:8080/] failed to finish deploying within the timeout period [30000]. The Deployable state is thus unknown.

File: optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/employee/EmployeeListPanel.java [hunk 1]
```diff
@@ -106,10 +104,10 @@ public void refresh(final @ForEvent("click") MouseEvent e) {
         }
         return promiseUtils.promise((res, rej) -> {
             EmployeeRestServiceBuilder.getEmployeeList(tenantStore.getCurrentTenantId(), FailureShownRestCallback
-                    .onSuccess(newEmployeeList -> {
-                        searchBar.setListToFilter(newEmployeeList);
-                        res.onInvoke(promiseUtils.resolve());
-                    }));
+                                                                                                                 .onSuccess(newEmployeeList -> {
```
Comment (rsynek) [inline optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/employee/EmployeeListPanel.java]: Please fix the formatting

File: optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/skill/SkillListPanel.java [hunk 2]
```diff
@@ -100,10 +98,10 @@ public void refresh(final @ForEvent("click") MouseEvent e) {
         }
         return promiseUtils.promise((res, rej) -> {
             SkillRestServiceBuilder.getSkillList(tenantStore.getCurrentTenantId(), FailureShownRestCallback
-                    .onSuccess(newSkillList -> {
-                        searchBar.setListToFilter(newSkillList);
-                        res.onInvoke(promiseUtils.resolve());
-                    }));
+                                                                                                           .onSuccess(newSkillList -> {
```
Comment (rsynek) [inline optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/skill/SkillListPanel.java]: ditto

File: optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/spot/SpotListPanel.java [hunk 3]
```diff
@@ -107,10 +105,10 @@ public void refresh(final @ForEvent("click") MouseEvent e) {
         }
         return promiseUtils.promise((res, rej) -> {
             SpotRestServiceBuilder.getSpotList(tenantStore.getCurrentTenantId(), FailureShownRestCallback
-                    .onSuccess(newSpotList -> {
-                        searchBar.setListToFilter(newSpotList);
-                        res.onInvoke(promiseUtils.resolve());
-                    }));
+                                                                                                         .onSuccess(newSpotList -> {
```
Comment (rsynek) [inline optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/spot/SpotListPanel.java]: ditto

**Comment by ge0ffrey** [comment]
jenkins retest this. This time it will save the server logs.

**Pull request merged**